### PR TITLE
[ja] Fixed some translations

### PR DESCRIPTION
**Pull request by NeetShin**
:3 :3

### CODE REVIEW

File: DireFloof/DireFloof/ja.xliff [hunk 1]
```diff
@@ -1899,7 +1899,7 @@ AmaroqはMastodonの愛あるアイデアで開発され、支えられていま
       </trans-unit>
       <trans-unit id="replied to">
         <source>replied to</source>
-        <target>リプライしました</target>
+        <target>さんがリプライしました</target>
```
Comment (53ningen) [inline DireFloof/DireFloof/ja.xliff]: may be`さんにリプライしました` ?

File: DireFloof/ja.lproj/Localizable.strings [hunk 2]
```diff
@@ -267,5 +267,5 @@ AmaroqはMastodonの愛あるアイデアで開発され、支えられていま
 "replied" = "さんがリプライしました";
 
 /* replied to */
-"replied to" = "リプライしました";
+"replied to" = "さんがリプライしました";
```
Comment (53ningen) [inline DireFloof/ja.lproj/Localizable.strings]: may be `さんにリプライしました ?`

**Comment by NeetShin** [comment]
Hm, that's one of difficult Japanese grammar but see this.

**Comment by 53ningen** [comment]
Oh, may be it is correct translation. Sorry 🙏

**Pull request merged**